pynamodb/connection/base.py

```diff
@@ -879,6 +893,8 @@ def put_item(self,
         """
         Performs the PutItem operation and returns the result
         """
+        self._check_condition('condition', condition, expected, conditional_operator)
```

Comment:
Yes, this is to render the name of the parameter in the error message. When we use this for query and scan filters, the parameter name will be different.